pkg/endpoint/policy.go

```diff
@@ -313,12 +311,7 @@ func (e *Endpoint) regenerateConsumable(owner Owner, labelsMap *identityPkg.Iden
 	}
 
 	if owner.AlwaysAllowLocalhost() || c.L4Policy.HasRedirect() {
-		if e.allowIngressIdentity(identityPkg.ReservedIdentityHost) {
-			e.getLogger().WithFields(logrus.Fields{
-				logfields.PolicyID: identityPkg.ReservedIdentityHost,
-			}).Debug("policy changed due to allowing host identity when it previously was not allowed")
-			changed = true
-		}
+		_ = e.allowIngressIdentity(identityPkg.ReservedIdentityHost)
```

Comment:
is the `_ =` needed for some side effect? I didn't think the compiler was that aggressive about dropping functions.

Reply:
That crossed my mind as the reason. It wasn't apparent because `allowIngressIdentity` returning anything wasn't obvious, nor that the return is important. Anyway, no worries.

Reply:
Indeed ! This function will be going away soon anyway ;)